src/acceptanceTest/java/uk/gov/hmcts/futurehearings/hmi/acceptance/common/header/dto/factory/PayloadHeaderDtoFactory.java

```diff
@@ -0,0 +1,94 @@
+package uk.gov.hmcts.futurehearings.hmi.acceptance.common.header.dto.factory;
+
+import io.restassured.http.Header;
+import io.restassured.http.Headers;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import lombok.experimental.Accessors;
+import uk.gov.hmcts.futurehearings.hmi.acceptance.common.header.dto.BusinessHeaderDto;
+import uk.gov.hmcts.futurehearings.hmi.acceptance.common.header.dto.SystemHeaderDto;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Accessors(fluent = true)
+@ToString
+@EqualsAndHashCode
+public class PayloadHeaderDtoFactory {
```

Comment:
This class can be declared as final class as it is having only private constructor

Reply:
Out of scope